src/Enum/RuleTips.php

```diff
@@ -7,7 +7,13 @@
 final class RuleTips
 {
     /**
+     * @todo this wont make sense anymore, as there will be a rule to make method private/protected
```

Comment:
even if we find a method which is only used locally, it could be usefull to annotate it with `@api` as it is meant for use by external callers.

I think therefore the `NARROW_SCOPE` rule-tip is not correct and we should still use `SOLUTION_MESSAGE` everywhere..?

Reply:
I think I'll remove these tips, as they depends on the context and cannot be bound to any rule. This confussion :)

Reply:
IMO the hints are really valuable